core/src/test/scala/org/apache/spark/sql/store/CreateIndexTest.scala

```diff
@@ -105,29 +110,30 @@ class CreateIndexTest extends SnappyFunSuite with BeforeAndAfterEach {
     snContext.sql(s"create index $indexThree on $tableName (COL1, COL3)")
     indexesToDrop += indexThree
 
-    val executeQ = QueryExecutor(snContext)
+    val executeQ = CreateIndexTest.QueryExecutor(snContext)
 
     executeQ(s"select * from $tableName where col1 = 111") {
-      validateIndex(Seq(indexOne))(_)
+      CreateIndexTest.validateIndex(Seq(indexOne))(_)
     }
 
     executeQ(s"select * from $tableName where col2 = 'aaa' ") {
-      validateIndex(Seq.empty, tableName)(_)
+      CreateIndexTest.validateIndex(Seq.empty, tableName)(_)
     }
 
     executeQ(s"select * from $tableName where col2 = 'bbb' and col3 = 'halo' ") {
-      validateIndex(Seq(indexTwo))(_)
+      CreateIndexTest.validateIndex(Seq(indexTwo))(_)
     }
 
     executeQ(s"select * from $tableName where col1 = 111 and col3 = 'halo' ") {
-      validateIndex(Seq(indexThree))(_)
+      CreateIndexTest.validateIndex(Seq(indexThree))(_)
     }
   }
 
   test("Test create Index on Column Table using Snappy API") {
     val tableName: String = "tcol1"
     val snContext = SnappyContext(sc)
     context.set(snContext)
+    System.setProperty(io.snappydata.Property.EnableExperimentalFeatures.configEntry.key, "true")
```

Comment:
The afterEach is also setting the conf property, and System property is not being cleared at the end of each test either. So if the intention was to test either of conf or System.property then this is not achieving it. Set either of the two ways in the test, then clear both at the end of each test.